src/MaxText/maxtext_utils.py

```diff
@@ -97,14 +99,14 @@ def get_shaped_batch(config):
 
 def should_prevent_cse_in_remat(config):
   """Determines whether to prevent common subexpression elimination (CSE) in remat.
-  
+
```

Comment:
nit: thanks for fixing whitespace but could this possibly be done in another PR? some of us use blame a lot so this can be helpful